src/modules/welcome/modules/getStarted/templates/createAccount.html

```diff
@@ -0,0 +1,69 @@
+<div class="create-account">
+
+    <div ng-click="$ctrl.back()" class="back">
+        НАВЗАД!
+    </div>
+
+    <div ng-class="{active: $ctrl.hasAccount}" class="created-animation">
+        <w-avatar address="$ctrl.address"></w-avatar>
+    </div>
+
+    <div class="block">
+
+        <p class="title text">Create Your Account</p>
+        <p class="text">Sign up for a fee account below.</p>
+
+        <hr/>
+
+        <p class="text">Choose your address avatar</p>
+
+        <div class="avatar-block">
+            <w-avatar ng-click="$ctrl.setActiveSeed(item)" ng-repeat="item in $ctrl.seedList"
+                      ng-class="{active: item.active}"
+                      address="item.address"></w-avatar>
+        </div>
+
+        <hr/>
+
+        <p class="text">Setup password</p>
+
+        <form name="createPassword" ng-submit="createPassword.$valid && $ctrl.next()" novalidate>
+
+            <section layout="row" layout-sm="column" layout-align="center center" layout-wrap>
+                <md-input-container class="md-block">
+                    <label>Password</label>
+                    <input name="password" type="password" required minlength="8"
+                           ng-pattern="/(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])[0-9a-zA-Z!@#$%^&*]/g"
```

Comment:
I'm not sure if it's a good practice. I guess we should keep our pattern collection somewhere. Can applying of all these limitations be done with a help of a directive? Something like:
```
<md-input-container class="md-block">
  <label>Password</label>
  <input name="password" w-apply-rules="password" />
</md-input-container>
```
so the rules (min and max length, regex, etc) are then seen by `mdInput` and applied there?